src/auro-calendar.js

```diff
@@ -198,6 +199,7 @@ export class AuroCalendar extends RangeDatepicker {
     ) {
       if (changedProperties.has("month") || changedProperties.has("year")) {
         this.monthChanged(this.month, this.year);
+        this.notifyMonthChanged(this.month, this.year, this.numCalendars);
```

Comment:
Probably best to call `this.notifyMonthChanged()` from `this.monthChanged()`.

Reply:
From my understanding, `monthChanged()` seems to be in the `RangeDatepicker` class from range-datepicker.js, and `this.numCalendars` is a property created in auro-calendar only. If we move it to  `monthChanged()` in range-datepicker.js, we wouldn't be able to pass the number of calendars in the event unless we change the parameters of the method to pass `this.numCalendars`. Let me know if you'd like to go down that route or keep it in auro-calendar.

Reply:
Eventually we want to re-write the 3rd party tool anyhow here. Just copy the entire function as is from `RangeDatepicker` and then modify it to include the call to `notifyMonthChanged()`.